recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - gridgen
+        - fiona
+        - python
+        - pandas
+        - numpy
+        - matplotlib
+        - pygridgen
+
+test:
+    imports:
+        - pygridtools
+    commands:
+        - nosetests
+    requires:
+        - nose
+
+about:
+    home: https://github.com/Geosyntec/pygridtools.git
+    license:  BSD License
```

Comment:
I default to the 3-clause BSD. My goals is that these are released under a permissive license.